Frontend/src/hooks/useUserData.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from '@tanstack/react-query'
+import { getCompetitorsInfo } from '../api/user/post/get_user_info'
+
+export function useUserData(ids: number[]) {
+  return useQuery({
+    queryFn: () => (ids.length > 0 ? getCompetitorsInfo(ids) : []),
```

Comment:
Should `getCompetitorsInfo` have `Users` rather than `Competitors` in the name, for consistency with everything else?
And for the hook, I can't decide whether `useUsersData` would be a better name, since it accepts multiple ids.

Reply:
+1, especially because you're using `user-info` as `queryKey` in the Tanstack-Query config a few lines down below